dashboard/app/controllers/plc_enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PlcEnrollmentEvaluationsController < ApplicationController
```

Comment:
why use Plc sometimes and spell out ProfessionalLearningCourses sometimes? Would it make sense to namespace all of these controllers?